### PR TITLE
chore: fix linting issues

### PR DESCRIPTION
**Pull request by behouba**
This PR fixes linting issues reported by the latest version of `golangci-lint`.

### CODE REVIEW

File: crit/explore.go [hunk 1]
```diff
@@ -328,7 +328,7 @@ func (c *crit) ExploreRss() ([]*RssMap, error) {
 		for _, pagemapEntry := range pagemapImg.Entries[1:] {
 			pagemapData := pagemapEntry.Message.(*pagemap.PagemapEntry)
 			rss := Rss{
-				PhyAddr:  fmt.Sprintf("%x", pagemapData.GetVaddr()),
+				PhyAddr:  strconv.FormatUint(pagemapData.GetVaddr(), 16),
```
Comment (rst0git) [inline crit/explore.go:331]: @behouba Would you be able to add more information in the commit message to explain why these changes are needed? For example, you can add any errors or warnings reported by golangci-lint.

Comment (behouba) [reply crit/explore.go:331]: Sure!

**Review by rst0git — APPROVED** [review]
